x/staking/client/cli/tx.go

```diff
@@ -229,23 +235,27 @@ func NewRedelegateCmd() *cobra.Command {
 			fmt.Sprintf(`Redelegate an amount of illiquid staking tokens from one validator to another.
 
 Example:
-$ %s tx staking redelegate %s1gghjut3ccd8ay0zduzj64hwre2fxs9ldmqhffj %s1l2rsakp388kuv9k8qzq6lrm9taddae7fpx59wm 100stake --from mykey
+$ %s tx staking redelegate cosmoss1gghjut3ccd8ay0zduzj64hwre2fxs9ldmqhffj cosmoss1l2rsakp388kuv9k8qzq6lrm9taddae7fpx59wm 100stake --from mykey
```

Comment:
should be validators addresses

Reply:
https://github.com/cosmos/cosmos-sdk/pull/17098/commits/425e2db24e9de26da75ce5f047253ba96b662c7f -> the second one is redelegate is as well a validator and should be updated.